tests/apps/courses/test_cms_plugins_blogpost.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for the BlogPost plugin and its model
+"""
+from django import forms
+from django.conf import settings
+
+from cms.api import add_plugin, create_page
+from cms.test_utils.testcases import CMSTestCase
+from djangocms_picture.cms_plugins import PicturePlugin
+
+from richie.apps.core.factories import FilerImageFactory, UserFactory
+from richie.apps.core.helpers import create_i18n_page
+from richie.apps.courses.cms_plugins import BlogPostPlugin
+from richie.apps.courses.factories import BlogPostFactory
+from richie.apps.courses.models import BlogPostPluginModel
+
+
+class BlogPostPluginTestCase(CMSTestCase):
+    """
+    Test that BlogPostPlugin correctly displays a BlogPost's page placeholders content
+    """
+
+    def test_cms_plugins_blogpost_form_page_choices(self):
+        """
+        The form to create a blogpost plugin should only list blogpost pages
+        in the select box.
+        """
+
+        class BlogPostPluginModelForm(forms.ModelForm):
+            """A form for testing the choices in the select box"""
+
+            class Meta:
+                model = BlogPostPluginModel
+                fields = ["page"]
+
+        blogpost = BlogPostFactory()
```

Comment:
Add this and the test will fail unless you add the clause I pointed above.
```suggestion
        blog_page = create_i18n_page(published=True)
        blogpost = BlogPostFactory(page_parent=blog_page)
```